tensorflow/python/tools/optimize_for_inference_test.py

```diff
@@ -751,7 +752,10 @@ def testFuseDecomposedBatchNorm_PatternMismatchCase(self):
     self.assertAllClose(original_result, optimized_result)
 
   @test_util.run_deprecated_v1
-  def testFuseDecomposedBatchNorm_FormatUnsupportCase(self):
+  def testFuseDecomposedBatchNorm_FormatUnsupportedCase(self):
+    if tf_config.list_physical_devices("CPU"):
```

Comment:
This will always be true, since there's always at least one CPU device (the host).

Plus, don't you want this to run on CPU?

Reply:
This test case is added to check - if the frozen graph has NCHW format, we are not going to fuse it. The data format check is just to make sure we don't handle NCHW case. If with the current TensorFlow, it won't even allow creating conv op with NCHW format for CPU, the code checking is just double guarantee.
Can we Not to add this NCHW data format test case?